_posts/2016-04-26-cloudready-linux-mint.markdown

```diff
@@ -0,0 +1,166 @@
+---
+layout: post
+title: Dual-Boot Cloudready and Linux Mint
+tags: linux-mint cloudready dual-boot grub
+username: likethemammal
+---
```

Comment:
Crossposts are used for referencing the same post on a different site. I don't have a problem using it for multiple things like this though.